src/vm/gdbjit.cpp

```diff
@@ -1330,30 +1365,47 @@ void NotifyGdb::MethodCompiled(MethodDesc* MethodDescPtr)
     }
     pCH->SetCalledMethods(NULL);
 
-    /* Get debug info for method from portable PDB */
-    HRESULT hr = GetDebugInfoFromPDB(MethodDescPtr, &symInfo, symInfoLen, locals);
-    if (FAILED(hr) || symInfoLen == 0)
-    {
-        return;
-    }
-
     MetaSig sig(MethodDescPtr);
     int nArgsCount = sig.NumFixedArgs();
     if (sig.HasThis())
         nArgsCount++;
 
-    method = new FunctionMember(MethodDescPtr, locals.size, nArgsCount);
-    // method return type
-    method->m_member_type = GetArgTypeInfo(MethodDescPtr, pTypeMap, 0);
-
     unsigned int firstLineIndex = 0;
     for (;firstLineIndex < symInfoLen; firstLineIndex++) {
         if (symInfo[firstLineIndex].lineNumber != 0 && symInfo[firstLineIndex].lineNumber != HiddenLine && symInfo[firstLineIndex].fileName[0] != 0) break;
     }
-    method->GetLocalsDebugInfo(pTypeMap, locals, symInfo[firstLineIndex].nativeOffset);
 
-    // method's class
-    GetTypeInfoFromTypeHandle(TypeHandle(method->md->GetMethodTable()), pTypeMap);
+    method = new (nothrow) FunctionMember*[method_count];
```

Comment:
Ah, I am sorry, I've missed the fact that the variable is global. Then please ignore my comment.